packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx

```diff
@@ -186,19 +285,32 @@ const AgendaItem = (props: Props) => {
           isDragging={isDragging}
           isUnsyncedFacilitatorStage={isUnsyncedFacilitatorStage}
         >
-          {content}
+          {isEditing ? (
+            <EditableText
+              autoFocus={isEditing}
+              error={error?.message}
+              handleSubmit={handleSubmit}
+              validate={validate}
+              maxLength={AGENDA_ITEM_MAX_CHARS}
+              initialValue={content}
+              placeholder={''}
+              hideIcon={isDisabled || !isHovering}
+              disabled={isDisabled}
+              onEditingChange={setIsEditing}
+            />
+          ) : (
+            content
+          )}
         </MeetingSubnavItem>
-        <DeleteIconButton
-          aria-label={'Remove this agenda topic'}
-          disabled={!!endedAt}
-          icon='cancel'
-          onClick={handleRemove}
-          palette='midGray'
-        />
       </AgendaItemStyles>
       {tooltipPortal(
-        pinned ? `Unpin "${content}" from every check-in` : `Pin "${content}" to every check-in`
+        endedAt
+          ? pinned
+            ? `Unpin "${content}" from every check-in`
+            : `Pin "${content}" to every check-in`
+          : `Select the action for "${content}"`
```

Comment:
-1 I don't think we need a tooltip for `Select the action` as I feel that the menu icon is clear enough. There's a risk of having too many tooltips and it becoming annoying for the user